source/PluginDev/Assets/GooglePlayGames/Platforms/Android/AndroidClient.cs

```diff
@@ -786,13 +795,22 @@ public void ShowLeaderboardUI(string leaderboardId, LeaderboardTimeSpan span, Ac
                 return;
             }
 
+            Action<UIStatus> wrapperCallback = status =>
+            {
+                if (status == UIStatus.NotAuthorized)
+                {
+                    SignOut();
+                }
```

Comment:
Can we move this wrappedCallback creation to private method, to not duplicate it?

Reply:
Done!